code/modules/antagonists/traitor/contractor/datums/objective_contract.dm

```diff
@@ -0,0 +1,235 @@
+			// Rooms
+			"Alternate Construction Area",
+			"Barber Shop",
+			"Escape Shuttle Hallway Podbay",
+			"Garden",
+			"Incinerator",
+			"Locker Room",
+			"Locker Toilets",
+			"Maintenance Bar",
+			"Medical Secondary Storage",
+			"Mechanic Workshop",
+			"Port Emergency Storage",
+			"Psych Room",
+			"Toxins Launch Room",
+			"Toxins Mixing Room",
+			"Turbine",
+			"Virology",
+			"Waste Disposal",
+			// Maintenance
+			"Aft Port Solar Maintenance",
+			"Aft Starboard Solar Maintenance",
+			"Arrivals North Maintenance",
+			"Bar Maintenance",
+			"Cargo Maintenance",
+			"Dormitory Maintenance",
+			"Electrical Maintenance",
+			"EVA Maintenance",
+			"Engineering Maintenance",
+			"Fore Port Solar Maintenance",
+			"Fore Starboard Solar Maintenance",
+			"Genetics Maintenance",
+			"Locker Room Maintenance",
+			"Medbay Maintenance",
+			"Science Maintenance",
+		),
+		EXTRACTION_DIFFICULTY_MEDIUM = list(
+			// Rooms
+			"Aft Primary Hallway",
+			"Atmospherics",
+			"Arcade",
+			"Assembly Line",
+			"Auxiliary Tool Storage",
+			"Break Room",
+			"Blueshield's Office",
+			"Cargo Bay",
+			"Chapel",
+			"Chapel Office",
+			"Clown's Office",
+			"Construction Area",
+			"Courtroom",
+			"Dormitory Toilets",
+			"Engineering",
+			"Engineering Control Room",
+			"Escape Shuttle Hallway",
+			"Experimentation Lab",
+			"Holodeck Alpha",
+			"Hydroponics",
+			"Library",
+			"Mime's Office",
+			"Mining Dock",
+			"Morgue",
+			"Office Supplies",
+			"Pet Store",
+			"Primary Tool Storage",
+			"Research Division",
+			"Security Checkpoint",
+			"Technical Storage",
+			"Teleporter",
+			"Toxins Storage",
+			"Vacant Office",
+			"Research Testing Lab",
+			"Xenobiology Lab",
+			// Maintenance
+			"Atmospherics Maintenance",
+			"Bridge Maintenance",
+		),
+		EXTRACTION_DIFFICULTY_HARD = list(
+			// No AI Chamber because I'm not that sadistic.
+			// Most Bridge areas are excluded because of they'd be basically impossible. So are Brig areas.
+			"AI Satellite Antechamber",
+			"AI Satellite Atmospherics",
+			"AI Satellite Service",
+			"AI Satellite Hallway",
+			"Bar",
+			"Cargo Office",
+			"Central Primary Hallway",
+			"Chemistry",
+			"Chief Engineer's office",
+			"Chief Medical Officer's office",
+			"Cloning Lab",
+			"Cryogenics",
+			"Dorms",
+			"Engineering Equipment Storage",
+			"Engineering Foyer",
+			"EVA Storage",
+			"Gateway",
+			"Genetics Lab",
+			"Gravity Generator",
+			"Head of Personnel's Office",
+			"Heads of Staff Meeting Room",
+			"Kitchen", // Chef CQC is no joke.
+			"Mech Bay",
+			"Medbay",
+			"Medbay Reception",
+			"Medical Storage",
+			"Medical Treatment Center",
+			"Medbay Patient Ward",
+			"Messaging Server Room",
+			"Mr Chang's",
+			"Nanotrasen Representative's Office",
+			"Paramedic",
+			"Port Primary Hallway",
+			"Quartermaster's Office",
+			"Research Director's Office",
+			"Research and Development",
+			"Robotics Lab",
+			"Surgery 1",
+			"Surgery 2",
```

Comment:
Feel these could be medium. Surgery rooms have shutters and opaque doors.

Reply:
True, but surgery is frequented most of the times and is close to the treatment center where most doctors are, therefore making an extraction in the ORs very risky